npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -136,19 +136,24 @@ export async function generateTsConfig (devServerConfig: AngularWebpackDevServer
     includePaths.push(...polyfills.map((p: string) => getProjectFilePath(workspaceRoot, p)))
   }
 
-  const cypressTypes = getProjectFilePath(workspaceRoot, 'node_modules', 'cypress', 'types', 'index.d.ts')
+  const typeRoots = [
+    getProjectFilePath(workspaceRoot, 'node_modules'),
+    getProjectFilePath(workspaceRoot, 'node_modules', 'types'),
+  ]
```

Comment:
I'm a bit confused here - wouldn't specifying a `node_modules` typeRoot include everything under that directory? If that's the case why does `node_modules/types` need to be specified? Also, are we really intending to target `node_modules/types` or is that supposed to be the default `@types`?

Reply:
Ahhh, yes. `node_modules/types` can be removed. The main issue is resolving the `typeRoots` to the actual projects `node_modules` not the temp dirs. This commit removes line 141. 67b86c8ef089f22335a2455b99292ba392eea2b5